float64 KP
float64 KI
int16 Ireset